app/webpacker/controllers/cookie-acceptance_controller.js

```diff
@@ -0,0 +1,32 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
```

Comment:
Can we add a test for this controller?

Reply:
Yes, my next task is to test all the stimulus controllers

Reply:
It would make sense to add it into this PR unless you're under pressure to get this shipped for a demo?